templates/application/conversation.html.twig

```diff
@@ -0,0 +1,64 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}
+	Répondre à une annonce
+{% endblock %}
+
+{% block body %}
+
+	<div class="container my-5">
+		<h2 class="mb-3">Répondre à l'annonce :
+			{{application.offer.title}}</h2>
+		{% include "offer/_offer_card.html.twig" with {offer: application.offer}  %}
+	</div>
+
+	<div class="container my-5">
+		{% block subject %}
+			<h2>votre Conversation concernant la candidature :
+			</h2>
+		{% endblock %}
+		<p>
+		Vous disctuez avec :
+		</p>
+		{{application.user}}<br>
+		<p>
+		Concernant l'offre :
+		</p>
+		{{application.offer.title}}<br>
+		<p>
+		Postée par :
+		</p>
+		{{application.offer.breeder.email }}<br>
+		{% for dog in application.dogs %}
+			<p>
+				Chien concernés :
+				{{ dog.name }}</p>
+		{% endfor %}
+		<br>
+		<h3>Derniers messages :</h3>
+		<div class="border border-bottom-0 rounded p-3 mb-2 bg-light text-dark ">
+			{% for message in application.messages %}
+				<div class="bg-body-secondary ">
+					<div class="fw-bold border rounded p-2 mb-2 border border-success p-2 " style="--bs-border-opacity: .5;">
+
+						{% if message.isSentByAdopter %}
+							{{ application.user.name }}
+						{% else %}
+							{{ application.offer.breeder.name }}
+						{% endif %}
+					</div>
+					<p>
+						{{ message.text }}</p>
+				</div>
+			{% endfor %}
+		</div>
+
+		{{form_start(form)}}
```

Comment:
Sur le GDoc on avait pas noté le formulaire de réponse au dessus des messages?

Reply:
j'ai pas trouver sur le GDoc , mais j'ai fait le changement